docs/townhall-history.md

```diff
@@ -2,10 +2,43 @@
 
 A list of previous Town Halls, their planned schedule, and the recording of the meeting.
 
+## 10/27/2022
+[Full YouTube video](https://youtu.be/B74WHxX5EMk)
+
+### Agenda
+
+- Conquer Data Governance with Acryl Data’s Metadata Tests - Learn how to tackle Data Governance with incremental, automation-driven governance using Metadata Tests provided in Acryl Data’s managed DataHub offering
+- Community Case Study - The Grab Team shares how they are using DataHub for data discoverability, automated classification and governance workflows, data quality observability, and beyond!
+- Upcoming Ingestion Sources - We’ll tell you the ins and outs of our upcoming dbt Cloud and Unity Catalog connectors
+- Sneak Peek! Saved Views - Learn how you can soon use Saved Views to help end-users navigate entities in DataHub with more precision and focus
+- Performance Improvements - Hear about the latest upgrades to DataHub performance
+
+## 9/29/2022
+[Full YouTube video](https://youtu.be/FjkNySWkghY)
+
+### Agenda
+
+- Column Level Lineage is here! - Demo of column-level lineage and impact analysis in the DataHub UI
+- Community Case Study - The Stripe Team shares how they leverage DataHub to power observability within their Airflow-based ecosystem
+- Sneak Peek! Automated PII Classification - Preview upcoming functionality to automatically identify data fields that likely contain sensitive data
+- Ingestion Improvements Galore - Improved performance and functionality for dbt, Looker, Tableau, and Presto ingestion sources
+
+## 8/25/2022
+[Full YouTube video](https://youtu.be/EJCKxKBvCwo)
+
+### Agenda
+
+- Community Case Study - The Etsy Team shares their journey of adopting DataHub
+- Looker <> DataHub Improvements - surface the most relevant Looks and Dashboards
```

Comment:
May want to remove this `<>` @maggiehays  as this seems to be breaking the docs build